src/main/java/graphql/GraphQL.java

```diff
@@ -127,6 +127,27 @@ public static Builder newGraphQL(GraphQLSchema graphQLSchema) {
         return new Builder(graphQLSchema);
     }
 
+    /**
+     * Helps you build a GraphQL object from a previous GraphQL object
+     *
+     * @param previousGraphQL the previous
+     *
+     * @return a builder of GraphQL objects
+     */
+    public static Builder newGraphQL(GraphQL previousGraphQL) {
```

Comment:
we also have the 'transform' pattern with a lambda as argument ... maybe we should do that? or both?

Reply:
Ahh yes - good call - will do

Reply:
done